src/JuMP.jl

```diff
@@ -659,6 +697,23 @@ function Base.copy(c::QuadConstraint, new_model::Model)
 end
 
 ##########################################################################
+# SOCConstraint is a second-order cone constraint of the form
+# ||Ax-b||₂ + cᵀx + d ≤ 0
+
+type NormConstraint{T} <: JuMPConstraint
```

Comment:
And finally here, so I can catch NormConstraints on Uncertains and do the rgiht thing with them.